cmd/discord_client/main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"haracho-go/internal/client/discord"
+	"haracho-go/internal/command"
+	"os"
+)
+
+func main() {
+	command.RegisterCommands()
+
+	c := discord.Client{Token: os.Getenv("TOKEN")}
```

Comment:
```suggestion
	collection := new(client.CommandCollection)
	command.RegisterCommands(collection)

	c := discord.Client{Token: os.Getenv("TOKEN"), Collection: collection}
```
別記の通りに `RegisterCommands` を直すとこうなる
`import` も追加してね